doc/static-route/SONiC_static_route_bfd_hld.md

```diff
@@ -0,0 +1,300 @@
+# Scope
+
+This document describes the design of using BFD to detect static route nexthop and based on BFD session state, installing, updating, and removing static route to the system.<br  /> 
```

Comment:
Did we explore FRR BFD data-plane offload capability? we push all static routes to FRR today, it would be ideal if you could get the BFD configs for a route/NH from FRR and handle in the SONiC.

Reply:
We did not explore FRR BFD offload yet. It could be the feature enhancement of FRR BFD. BfdOrch (inside Orchagent) is a service in the SONiC system, FRR BFD can use that service.